g11n-ws/vip-manager-i18n/src/test/java/com/vmware/vip/i18n/api/v2/translation/TranslationProductComponentAPITest.java

```diff
@@ -0,0 +1,92 @@
+    private static String MultComponentTranslationAPIURI3="/i18n/api/v2/translation/products/SampleProject/versions/1.0.0/multlocales/en,zh-Hants/multcomponents/component1?pseudo=false";
+    private static String MultComponentTranslationAPIURI4="/i18n/api/v2/translation/products/SampleProject/versions/1.0.0/multlocales/zh-Hants/multcomponents/component1,component2?pseudo=false";
+    
+    
+    @Before
+    public void setup() throws Exception {
+        String authenticationResult=RequestUtil.sendRequest(webApplicationContext,ConstantsForTest.POST, ConstantsForTest.AuthenticationAPIURI);
+        CacheUtil.cacheSessionAndToken(webApplicationContext, authenticationResult);
+    }
+
+    @Test
+    public void testSingleComponent() throws Exception {
+      String json = RequestUtil.sendRequest(webApplicationContext,ConstantsForTest.GET, SingleComponentTranslationAPIURI);
+        @SuppressWarnings("unchecked")
+        Map<String, Object> dataMap = (Map<String, Object>) JSONUtils.getMapFromJson(json).get("response");
+        long code = (long) dataMap.get("code");
+        Assert.assertTrue(code==200L);
+       
+    }
+
+    /**
+     * * @throws Exception
+     */
+    @Test
+    public void testMultiComponents1() throws Exception {
+        String json = RequestUtil.sendRequest(webApplicationContext,ConstantsForTest.GET, MultComponentTranslationAPIURI1);
+        @SuppressWarnings("unchecked")
+        Map<String, Object> dataMap = (Map<String, Object>) JSONUtils.getMapFromJson(json).get("response");
+        long code = (long) dataMap.get("code");
+        Assert.assertTrue(code==200L);
+    }
+
+    @Test
+    public void testMultiComponents2() throws Exception {
+        String json = RequestUtil.sendRequest(webApplicationContext,ConstantsForTest.GET, MultComponentTranslationAPIURI2);
+        @SuppressWarnings("unchecked")
+        Map<String, Object> dataMap = (Map<String, Object>) JSONUtils.getMapFromJson(json).get("response");
+        long code = (long) dataMap.get("code");
+        Assert.assertTrue(code==404L);
+    }
+    
+    @Test
+    public void testMultiComponents3() throws Exception {
+        String json = RequestUtil.sendRequest(webApplicationContext,ConstantsForTest.GET, MultComponentTranslationAPIURI3);
+        @SuppressWarnings("unchecked")
+        Map<String, Object> dataMap = (Map<String, Object>) JSONUtils.getMapFromJson(json).get("response");
+        long code = (long) dataMap.get("code");
+        Assert.assertTrue(code==404L);
+    }
+    @Test
+    public void testMultiComponents4() throws Exception {
+        String json = RequestUtil.sendRequest(webApplicationContext,ConstantsForTest.GET, MultComponentTranslationAPIURI4);
+        @SuppressWarnings("unchecked")
+        Map<String, Object> dataMap = (Map<String, Object>) JSONUtils.getMapFromJson(json).get("response");
+        long code = (long) dataMap.get("code");
+        Assert.assertTrue(code==404L);
+    }
```

Comment:
What behavior are these three cases used to test? Had better add comment for them.

Reply:
all case will response 404  except all part of requested translation exists